js/particle-modeler/index.js

```diff
@@ -226,9 +225,35 @@ export default class Interactive extends PureComponent {
     let newState = {};
     newState[prop] = {...this.state[prop]};
     newState[prop].value = parseToPrimitive(value);
+
+    if (prop === "elements") {
+      this.changeElementCount(value);
+    }
+
     this.setState(newState);
   }
 
+  changeElementCount(newElementCount) {
+    // has the number of elements been increased
+    if (newElementCount > this.state.elements.value) {
+      this.addNewDraggableAtom(newElementCount - 1, true);
```

Comment:
Note this causes a problem if the author clicks from 1 element directly on to the end of the slider, making it pop straight to 3 elements. The three images appear, but they are broken because only one new element was added.